src/lib/InputLabel.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+	let forName = '';
+	export { forName as for };
+	let className = '';
+	export { className as class };
```

Comment:
Just a minor note that you can combined these
```suggestion
	let forName = '';
	let className = '';
	export { forName as for, className as class };
```